scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -3035,6 +3256,74 @@ def validate_d16_quantized_softmax_receipt(summary: Any) -> None:
             raise AttentionKvRouteSelectorError(f"d16 quantized Softmax {key} drift")
 
 
+def validate_d16_two_head_quantized_softmax_receipt(summary: Any) -> None:
```

Ruff (0.15.12)

[warning] Too many branches (25 > 12)

(PLR0912)